aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
+
```

Comment:
Iff we change the section headers, as described in another comment, then we could, **optionally**, add a paragraph like,

> The following section describes the use of `google.type.Interval`.

**or** amend the first paragraph *slightly* to start,

> The following section describes the use of the [`google.type.Interval`][interval] type, found amongst the common protos that are described in [AIP-213][]. This type represents a range between....

Reply:
Yes, that would be fine. I'll wait until we've decided on everything else before making a change.

Reply:
Done (locally)